lms/envs/common.py

```diff
@@ -1916,6 +1916,7 @@
 INSTALLED_APPS = (
     # Standard ones that are always installed...
     'django.contrib.auth',
+    'username_length',
```

Comment:
Is there a reason to place this new app so high, and not near other common apps, at line 1959?

Reply:
I'm monkey patching the User model and I believe that it should be done the earlier the better to give ability other apps to use the change, that was the reason? If you think it could be another place - please advise the correct one and I'll change it with pleasure :).